frontend/src/app/commonComponents/Dropdown.tsx

```diff
@@ -82,8 +82,7 @@ const Dropdown: React.FC<Props & SelectProps> = ({
           {hintText && <span className="usa-hint">{hintText}</span>}
           <select
             className={classnames(
-              "usa-select",
-              selectClassName,
+              selectClassName || "usa-select",
```

Comment:
I think this might change the state select dropdown in our Facility edit page
prior:
<img width="1552" alt="Screen Shot 2022-11-29 at 11 47 00 AM" src="https://user-images.githubusercontent.com/6401323/204633173-ce277b82-3b0e-490a-a7ff-7200a5e87b70.png">
new:
<img width="1552" alt="Screen Shot 2022-11-29 at 11 46 35 AM" src="https://user-images.githubusercontent.com/6401323/204633208-da711e60-9a4c-4366-b37e-40b605ec8182.png">

Reply:
Very keen eye 🕵️‍♀️ it looks like this facility form field is the only select component that sets a `selectClassName` property, which is what was causing the style collisions between the various CSS classes applied here. 

Resolved in [da7dec6](https://github.com/CDCgov/prime-simplereport/pull/4759/commits/da7dec678100b845517af59148377cba0fd4f906).